File: LeetCode/125.js
/**
 * @param {string} s
 * @return {boolean}
 */
var isPalindrome = function(s) {
    let res = "";
     for(let i=0; i<s.length; i++){
     if(s[i]!==" "){
           res = res + s[i];
     }
     return res[i];
     }
     console.log(res);
    for(let j=s.length-1; j>0; j--){
        res = res+s[j];
    }
    if(res===s) return true;
    else return false;
};
console.log(isPalindrome("m om"));

File: LeetCode/125.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string word = " ";
        for(char ch : s){
          if(isalnum(ch)){
            if(ch >='A' && ch<='Z'){
                 ch += 32;
            }
            word += ch;
          }
        }
          int i = 0;
          int j = word.length() - 1;
          while(i < j){
            if(word[i] != word[j]){
                  return false;
            }
            i++;
            j--;
          }
        return true;
    }
};